### PR TITLE
Feat/add kv forsql

### PR DESCRIPTION
**Pull request by Trishisingh**

### CODE REVIEW

File: build/azDevOps/azure/air-infrastructure-data.yml
Comment (satenderrathee) [inline build/azDevOps/azure/air-infrastructure-data.yml:1]: In other PR for KV and ADLS ,Rhys has used variable group amido-stacks-euw-de-nonprod-network below values
TF_VAR_subnet_name: 'amido-stacks-euw-de-nonprod-pe'
TF_VAR_vnet_name: 'amido-stacks-euw-de-nonprod'
TF_VAR_vnet_resource_group_name: 'amido-stacks-euw-de-nonprod-network'

**Review by RhysBushnell — APPROVED** [review]

File: build/azDevOps/azure/air-infrastructure-data.yml [hunk 1]
```diff
@@ -116,7 +116,7 @@ stages:
                     TF_VAR_build_agent_ip: $(build_agent_ip)
                     TF_VAR_subnet_name: $(pe_subnet_name)
                     TF_VAR_vnet_name: $(pe_vnet_name)
-                    TF_VAR_vnet_resource_group_name: $(pe_resource_group_name)
+                    TF_VAR_vnet_resource_group_name: $(pe_resource_group_name
```
Comment (RhysBushnell) [inline build/azDevOps/azure/air-infrastructure-data.yml:119]: looks like a bracket has been removed

Comment (Trishisingh) [reply build/azDevOps/azure/air-infrastructure-data.yml]: thanks good spot

**Review by RhysBushnell — APPROVED** [review]

**Pull request merged**